python/ray/tune/progress_reporter.py

```diff
@@ -356,6 +352,12 @@ def _progress_str(
             )
 
         if has_verbosity(Verbosity.V1_EXPERIMENT):
+            if self._sort_by_metric and (self._metric is None or self._mode is None):
+                self._sort_by_metric = False
```

Comment:
shouldn't this always happen regardless of verbosity?

Reply:
I put it here, because I think this parameter is related only to trials table generated by `trial_progress_str` placed in this verbosity `if` statement. What do you think?

Reply:
I moved it
21483f1